### PR TITLE
Updates for linux docs

### LINKED ISSUE #36647: Handle linux distro release updates
It's been a while since the installer docs were looked at for the latest distro releases and updates. This issue is to track those updates.

#35754
#36640
#34873
#36658
#33717

---
[Associated WorkItem - 147247](https://dev.azure.com/msft-skilling/Content/_workitems/edit/147247)

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

- Added examples of installing dependencies to the linux articles
- Added some missing dependencies
- Clarify installing side-by-side versions of .NET during manual install
- Remove Snap (per discussion with product group)
- Ubuntu feed supports Arm64 now

Fixes #36647

---

#### Internal previews

<details><summary><strong>Toggle expand/collapse</strong></summary><br/>

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/install/linux-debian.md](https://github.com/dotnet/docs/blob/4583e4456f811ff650e093a6eabd5e29162f1346/docs/core/install/linux-debian.md) | [Install the .NET SDK or the .NET Runtime on Debian](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-debian?branch=pr-en-us-36686) |
| [docs/core/install/linux-opensuse.md](https://github.com/dotnet/docs/blob/4583e4456f811ff650e093a6eabd5e29162f1346/docs/core/install/linux-opensuse.md) | [Install .NET on openSUSE](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-opensuse?branch=pr-en-us-36686) |
| [docs/core/install/linux-scripted-manual.md](https://github.com/dotnet/docs/blob/4583e4456f811ff650e093a6eabd5e29162f1346/docs/core/install/linux-scripted-manual.md) | [Install .NET on Linux by using an install script or by extracting binaries](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-scripted-manual?branch=pr-en-us-36686) |
| [docs/core/install/linux-sles.md](https://github.com/dotnet/docs/blob/4583e4456f811ff650e093a6eabd5e29162f1346/docs/core/install/linux-sles.md) | [docs/core/install/linux-sles](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-sles?branch=pr-en-us-36686) |
| [docs/core/install/linux-ubuntu-1604.md](https://github.com/dotnet/docs/blob/4583e4456f811ff650e093a6eabd5e29162f1346/docs/core/install/linux-ubuntu-1604.md) | [Install .NET SDK or .NET Runtime on Ubuntu 16.04](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu-1604?branch=pr-en-us-36686) |
| [docs/core/install/linux-ubuntu-1804.md](https://github.com/dotnet/docs/blob/4583e4456f811ff650e093a6eabd5e29162f1346/docs/core/install/linux-ubuntu-1804.md) | [Install .NET on Ubuntu 18.04](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu-1804?branch=pr-en-us-36686) |
| [docs/core/install/linux-ubuntu-2004.md](https://github.com/dotnet/docs/blob/4583e4456f811ff650e093a6eabd5e29162f1346/docs/core/install/linux-ubuntu-2004.md) | [Install .NET SDK or .NET Runtime on Ubuntu 20.04](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu-2004?branch=pr-en-us-36686) |
| [docs/core/install/linux-ubuntu-2204.md](https://github.com/dotnet/docs/blob/4583e4456f811ff650e093a6eabd5e29162f1346/docs/core/install/linux-ubuntu-2204.md) | [docs/core/install/linux-ubuntu-2204](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu-2204?branch=pr-en-us-36686) |
| [docs/core/install/linux-ubuntu-2210.md](https://github.com/dotnet/docs/blob/4583e4456f811ff650e093a6eabd5e29162f1346/docs/core/install/linux-ubuntu-2210.md) | [Install .NET SDK or .NET Runtime on Ubuntu 22.10](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu-2210?branch=pr-en-us-36686) |
| [docs/core/install/linux-ubuntu-2304.md](https://github.com/dotnet/docs/blob/4583e4456f811ff650e093a6eabd5e29162f1346/docs/core/install/linux-ubuntu-2304.md) | [Install .NET on Ubuntu 23.04](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu-2304?branch=pr-en-us-36686) |
| [docs/core/install/linux-ubuntu.md](https://github.com/dotnet/docs/blob/4583e4456f811ff650e093a6eabd5e29162f1346/docs/core/install/linux-ubuntu.md) | [Install the .NET SDK or the .NET Runtime on Ubuntu](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu?branch=pr-en-us-36686) |
| [docs/core/tools/global-tools.md](https://github.com/dotnet/docs/blob/4583e4456f811ff650e093a6eabd5e29162f1346/docs/core/tools/global-tools.md) | [docs/core/tools/global-tools](https://review.learn.microsoft.com/en-us/dotnet/core/tools/global-tools?branch=pr-en-us-36686) |

</details>

### CODE REVIEW

File: docs/core/install/includes/linux-rpm-install-dependencies.md [hunk 1]
```diff
@@ -6,7 +6,13 @@ When you install with a package manager, these libraries are installed for you.
...
 - zlib
 
-If the target runtime environment's OpenSSL version is 1.1 or newer, you'll need to install **compat-openssl10**.
+If the target runtime environment's OpenSSL version is 1.1 or newer, you'll need to install `compat-openssl10`.
+
+Dependencies can be installed with the `yum install` command. The following snippet demonstrates installing the `libicu` library:
+
+```bash
+sudo yum install libc6
+```
```
Comment (tdykstra) [inline docs/core/install/includes/linux-rpm-install-dependencies.md:15]: should libc6 be libicu or vice versa?

File: docs/core/install/linux-scripted-manual.md [hunk 2]
```diff
@@ -126,7 +126,14 @@ Download a **binary** release for either the SDK or the runtime from one of the
...
 
-Alternatively, after downloading the .NET binary, the following commands may be run from the directory where the file is saved to extract the runtime. Running the following commands makes the .NET CLI commands available at the terminal and sets the required environment variables. **Remember to change the `DOTNET_FILE` value to the name of the downloaded binary**:
+Different versions of .NET can be extracted to the same folder, which coexist side-by-side.
+
+### Example
+
+The following commands set the environment variable `DOTNET_ROOT` to the current working directory followed by `.dotnet`. It then creates the directory if it doesn't exist and extracts the contents of the file specified by the `DOTNET_FILE` environment variable to the `.dotnet` directory. Both the `.dotnet` directory and its `tools` subdirectory are added to the `PATH` environment variable.
```
Comment (tdykstra) [inline docs/core/install/linux-scripted-manual.md:133]:
```suggestion
The following commands set the environment variable `DOTNET_ROOT` to the current working directory followed by `.dotnet`. They then create the directory if it doesn't exist and extract the contents of the file specified by the `DOTNET_FILE` environment variable to the `.dotnet` directory. Both the `.dotnet` directory and its `tools` subdirectory are added to the `PATH` environment variable.
```

File: docs/core/install/linux-ubuntu.md [hunk 3]
```diff
@@ -8,15 +8,15 @@ ms.date: 05/24/2023
 
 # Install the .NET SDK or the .NET Runtime on Ubuntu
 
-This article describes how to install .NET on Ubuntu. The Microsoft package repository contains every version of .NET that is currently, or was previously, supported on Ubuntu. Starting with Ubuntu 22.04, some versions of .NET are available in the Ubuntu package feed. For more information about available versions, see the [Supported distributions](#supported-distributions) section.
+This article describes how t o install .NET on Ubuntu. The Microsoft package repository contains every version of .NET that is currently, or was previously, supported on Ubuntu. Starting with Ubuntu 22.04, some versions of .NET are available in the Ubuntu package feed. For more information about available versions, see the [Supported distributions](#supported-distributions) section.
```
Comment (tdykstra) [inline docs/core/install/linux-ubuntu.md:11]:
```suggestion
This article describes how to install .NET on Ubuntu. The Microsoft package repository contains every version of .NET that is currently, or was previously, supported on Ubuntu. Starting with Ubuntu 22.04, some versions of .NET are available in the Ubuntu package feed. For more information about available versions, see the [Supported distributions](#supported-distributions) section.
```

**Review by tdykstra — APPROVED** [review]
Looks good, a few suggestions.

File: docs/core/install/includes/linux-rpm-install-dependencies.md (hunk 1 above)
Comment (adegeo) [reply docs/core/install/includes/linux-rpm-install-dependencies.md:15]:
```suggestion
Dependencies can be installed with the `yum install` command. The following snippet demonstrates installing the `libicu` library:

```bash
sudo yum install libicu
```
```

**Comment by adegeo** [comment]
Thanks tom!

**Pull request merged**